### DISCUSSION TITLE
Hard Rock shouldn't be incompatible with Mirror

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#19204] [Q&A]
Personally, I feel that most of Hard Rock's difficulty comes from the increased CS/HP/OD/AR rather than the map being flipped, so I think that players playing with Hard Rock should be able to use Mirror if they want to. Mirror's main uses come down to either overcoming mindblock on a map or trying to overcome biases in object placement, both of which can still be problems when playing with Hard Rock. Given that externally flipping your monitor seems to be allowed to overcome things such as mindblock and players have set scores by doing this, I think players should be able to use Hard Rock with Mirror in-game.

**Comment by frenzibyte** [comment]
So combine Hard Rock with Mirror to allow playing with increaesd beatmap diff without mirroring the beatmap? That sounds counterintuitive. I would rather have Hard Rock not mirror the beatmap rather than have this sort of combination.

**Reply by Flamiii** [reply]
I wouldn't be opposed to having Hard Rock not flip the map so that Hard Rock and Mirror can be used together, but I also feel that changing how Hard Rock fundamentally works raises a lot of questions. If this were to happen, what would happen to old Hard Rock scores and their replays? Would osu! stable and osu! lazer players be playing slightly different versions of the same mod when using Hard Rock? I totally get where you're coming from but I feel like it would be kind of complicated to change how Hard Rock fundamentally works and I'm not sure if it's really worth the trouble.
